x/evm/keeper/baseKeeper.go

```diff
@@ -58,21 +58,15 @@ func (k BaseKeeper) ForChain(chain string) types.ChainKeeper {
 	}
 }
 
-// SetPendingChain stores the chain pending for confirmation
-func (k BaseKeeper) SetPendingChain(ctx sdk.Context, chain nexus.Chain, p types.Params) {
+func (k BaseKeeper) setPendingChain(ctx sdk.Context, chain nexus.Chain, p types.Params) {
```

Comment:
I was doing that initially, but then I thought it would be less error-prone this way, since it keeps the original code. What if we had a few TODOs to remember us to remove these in the next release?